### PR TITLE
Adding dataset for 2023-04-25

### PR DESCRIPTION
**Pull request by tracykteal**
LondonMarathon dataset from @nrennie. Issue https://github.com/rfordatascience/tidytuesday/issues/540

### CODE REVIEW

File: data/2023/2023-04-25/post_vars.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "article_link": "https://nrennie.rbind.io/blog/web-scraping-rvest-london-marathon/",
+  "pic1_alt": "Hex logo of LondonMarathon R package. Background of the hex is an ariel photo of the Thames River and downtown London, and LondonMarathon is written on it, overlaid in red font.",
```
Comment (jonthegeek) [inline data/2023/2023-04-25/post_vars.json:3]:
```suggestion
  "pic1_alt": "Hex logo of LondonMarathon R package. Background of the hex is an aerial photo of the Thames River and downtown London, and LondonMarathon is written on it, overlaid in red font.",
```

**Review by jonthegeek — APPROVED** [review]
Just one typo that I'll commit as soon as I comment. I had to google to get that spelling right!

**Review by jonthegeek — APPROVED** [review]
Ok, that'll do!

File: data/2023/2023-04-25/post_vars.json (hunk 1 above)
Comment (tracykteal) [reply data/2023/2023-04-25/post_vars.json]: Thank you! Indeed I was not referring to the mermaid.

**Pull request merged**